src/scripts/slurm_status.py

```diff
@@ -388,15 +388,12 @@ def get_slurm_location(program):
     if _slurm_location_cache is not None:
         return os.path.join(_slurm_location_cache, program)
 
-    cmd = 'echo "%s/%s"' % (config.get('slurm_binpath'), 'scontrol')
```

Comment:
Was this some kind of sick joke?

Reply:
Don't look at the `git blame` :scream: !